android-client-sdk/src/main/java/com/devcycle/sdk/android/util/DVCSharedPrefs.kt

```diff
@@ -68,6 +68,25 @@ class DVCSharedPrefs(context: Context) {
         return null
     }
 
+    fun getString(key: String): String? {
+        val stringValue = preferences.getString(key,null)
+        if (stringValue == null) {
+            Timber.e("%s could not be found in SharedPreferences file: %s", key, R.string.cached_data)
```

Comment:
considering we'll be using this even when it's okay for the value to be null, maybe we should switch this to a debug/info log rather than an error (I believe that'd just Timber.d or Timber.i)